lib/ra11y/site.rb

```diff
@@ -3,7 +3,7 @@ class Site
 
     attr_accessor :path
 
-    def initialize(path)
+    def initialize(path, options: nil)
       @path = File.expand_path(path, Dir.pwd)
```

Comment:
Yes, pass in the initializer, but at least right now, options live on the `Ra11y` object, not `Ra11y::Site`. (That might be wrong)